src/include/common/csv_reader/csv_reader.h

```diff
@@ -13,14 +13,14 @@ class logger;
 namespace kuzu {
 namespace common {
 
-struct CSVReaderConfig {
-    CSVReaderConfig()
-        : escapeChar{CopyCSVConfig::DEFAULT_ESCAPE_CHAR},
-          tokenSeparator{CopyCSVConfig::DEFAULT_TOKEN_SEPARATOR},
-          quoteChar{CopyCSVConfig::DEFAULT_QUOTE_CHAR},
-          listBeginChar{CopyCSVConfig::DEFAULT_LIST_BEGIN_CHAR},
-          listEndChar{CopyCSVConfig::DEFAULT_LIST_END_CHAR},
-          hasHeader{CopyCSVConfig::DEFAULT_HAS_HEADER} {}
+struct ReaderConfig {
+    ReaderConfig()
+        : escapeChar{CopyConfig::DEFAULT_ESCAPE_CHAR},
+          tokenSeparator{CopyConfig::DEFAULT_TOKEN_SEPARATOR},
```

Comment:
I think this field should be renamed to "delimiter" and the CopyConfig::DEFAULT_TOKEN_SEPARATOR -> CopyConfig::DEFAULT_DELIMITER (though I think I had a comment to make these constants related to CSV have the word "CSV" in them).